scripts/maintainer.requirements.txt

```diff
@@ -8,3 +8,6 @@ clang
 # For building some test vectors
 pycryptodomex
 pycryptodome-test-vectors
+
+# For building `tests/data_files/server9-bad-saltlen.crt`
```

Comment:
Not just for building, also for running `check-python-files`. So this needs to be in `ci.requirements.txt`